examples/src/main/AndroidManifest.xml

```diff
@@ -95,6 +95,11 @@
         android:label="@string/title_multileg_route">
     </activity>
 
+      <activity
+          android:name=".RouteRefreshActivity"
+          android:label="@string/title_route_refresh">
+      </activity>
```

Comment:
Please move this to the QA app. We're overdue on removing the `examples` app but we shouldn't be adding new activities to it.

Reply:
I'll cut a separate PR for this.

Reply:
Moved since I had to fix the static analysis anyway.